src/tests/verify_resources.py

```diff
@@ -436,6 +439,26 @@ def test_14_res_contacts_match(rgs, rgfns, contacts):
 
     return errors
 
+def test_15_FACILITY_SITE_check():
+    # verify the required FACILITY.yaml and SITE.yaml files
+    errors = 0
+
+    groups = glob.glob(_topdir + '/topology/*')
```

Comment:
Note that we do a `os.chdir(_topdir + "/topology")` earlier, so all `glob()` calls can start relative to the `topology` dir, rather than needing to specify `_topdir` and `/topology/` as part of the glob path.

(In fact, including `_topdir`, which the program does not control, in the argument to `glob()` is technically unsafe, as the path could contain glob characters ...)